CHANGES.rst

```diff
@@ -1,6 +1,10 @@
 1.3 (unreleased)
 ================
 
+- Overhaul of the registry.regsearch as discussed in 
+  https://blog.g-vo.org/towards-data-discovery-in-pyvo.html.  This
+  should be backwards-compatible. [#278, #238, #176]
```

Comment:
the changelog usually includes the number of the PR adding a feature rather than the issue number